util/options.cc

```diff
@@ -92,7 +92,8 @@ Options::Options()
       allow_mmap_writes(true),
       is_fd_close_on_exec(true),
       skip_log_error_on_recovery(false),
-      stats_dump_period_sec(3600),
+    //   stats_dump_period_sec(3600),
+      stats_dump_period_sec(60), //1min
```

Comment:
Should be 10 min:
https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h#L614